### DISCUSSION TITLE
discord

### DISCUSSION THREAD

**Discussion by CFarketmez** [IBAX-io/go-ibax#475] [Ideas] [answered]
### Clear and concise description of the problem

on discord channel, there is so many advertisement link and scammer link. 

### Suggested solution

 discord admins must be online more.  admins have to take control discord channel.

### Validations

- [X] Read the [docs](https://docs.ibax.io).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by HalloKyle** [comment, marked as answer]
Thank you 4 ur support. Welcome to join us as Discord Moderator please contact Elunewhisper via Discord
